tools/src/main/java/org/apache/kafka/clients/tools/VerifiableLog4jAppender.java

```diff
@@ -88,6 +88,32 @@ private static ArgumentParser argParser() {
             .metavar("ACKS")
             .help("Acks required on each produced message. See Kafka docs on request.required.acks for details.");
 
+        parser.addArgument("--security-protocol")
```

Comment:
I think mostly the tools have been extended for security by taking a config file, e.g. `--producer.config` (see for example `VerifiableProducer`) instead of several command line options

I think this approach will be a little easier to extend to sasl as well